spec/features/admin_can_view_individual_orders_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+RSpec.describe 'Admin', type: :feature do
+  create_category
+  create_brand
+  create_items
+  create_user
+  create_rental
+  create_admin
+  feature 'admin' do
+    it 'can view individual order info' do
+      create_rental_items
+
+      visit login_path
+
+      fill_in "Username", with: 'admin'
+      fill_in "Password", with: 'admin'
+
+      click_button "Login"
+
+      click_link rental_one.id
+      within("#rental_item_row_#{rental_one.items.first.id}") do
+        expect(page).to have_content('70-200 Canon L')
+        expect(page).to have_content("1")
+        expect(page).to have_content("12")
+      end
+      expect(page).to have_content("Ross Edfort")
+      expect(page).to have_content('$120')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.